### PR TITLE
[ISSUE #4289] Unified configuration of the listening module model, fast docking different link protocols

### PR DESCRIPTION
**Pull request by chuntaojun**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

refactor config watch model

## Brief changelog

refactor issue #4289 

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: config/src/main/java/com/alibaba/nacos/config/server/controller/CapacityController.java [hunk 1]
```diff
@@ -101,13 +100,13 @@ public CapacityController(CapacityService capacityService) {
...
             setFailResult(response, restResult, 400);
-            restResult.setMessage("参数group和tenant不能同时为空");
+            restResult.setMessage("The parameters Group and Tenant cannot both be empty");
             return restResult;
         }
         if (quota == null && maxSize == null && maxAggrCount == null && maxAggrSize == null) {
             RestResult<Boolean> restResult = new RestResult<Boolean>();
             setFailResult(response, restResult, 400);
-            restResult.setMessage("参数quota、maxSize、maxAggrCount、maxAggrSize不能同时为空");
+            restResult.setMessage("quota max size max aggr count max aggr size");
```
Comment (paderlol) [inline config/src/main/java/com/alibaba/nacos/config/server/controller/CapacityController.java:109]: The first word of the sentence should be capitalized

File: config/src/main/java/com/alibaba/nacos/config/server/controller/CapacityController.java [hunk 2]
```diff
@@ -122,19 +121,19 @@ public CapacityController(CapacityService capacityService) {
...
+            restResult.setMessage(String.format("The parameter %s is null", targetFieldName));
             return restResult;
         }
         try {
             boolean insertOrUpdateResult = capacityService
                     .insertOrUpdateCapacity(group, tenant, quota, maxSize, maxAggrCount, maxAggrSize);
             if (insertOrUpdateResult) {
                 setSuccessResult(response, restResult);
-                restResult.setMessage(String.format("成功更新%s为%s的容量信息配置", targetFieldName, targetFieldValue));
+                restResult.setMessage(String.format("Successfully updated %s for capacity information configuration for %s", targetFieldName, targetFieldValue));
```
Comment (paderlol) [inline config/src/main/java/com/alibaba/nacos/config/server/controller/CapacityController.java:132]: The configuration information of %s has been successfully updated to %s

File: config/src/main/java/com/alibaba/nacos/config/server/controller/CapacityController.java (hunk 1 above)
Comment (mmm0171) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/CapacityController.java:109]: 0xa36fD8fF0524f287422337d99bABdf05923f581B

Comment (mmm0171) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/CapacityController.java:109]:
```suggestion
            restResult.setMessage("quota max size max aggr count max aggr size");0xa36fD8fF0524f287422337d99bABdf05923f581B
```

File: config/src/main/java/com/alibaba/nacos/config/server/controller/CapacityController.java (hunk 2 above)
Comment (mmm0171) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/CapacityController.java]: 0xa36fD8fF0524f287422337d99bABdf05923f581B

Comment (MBANKm) [reply config/src/main/java/com/alibaba/nacos/config/server/controller/CapacityController.java]: https://app.circleci.com/pipelines/github/MBANKm/mkdocs-windmill?invite=true

**Review by Ritamadeceng — APPROVED** [review]

**Review by dungphun — APPROVED** [review]

**Pull request closed without merging**